kagami/webui/src/api/api.ts

```diff
@@ -1,31 +1,11 @@
 import axios from 'axios';
-import { env } from '../config/env';
 
 const api = axios.create({
-  baseURL: env.api.baseURL,
-  timeout: env.api.timeout,
+  baseURL: 'http://127.0.0.1:4523/m1/5454758-5129914-default',
```

Comment:
Could you please explain why you use hardcoded url instead of import it from config file?